_topic_maps/_topic_map.yml

```diff
@@ -25,6 +25,8 @@ Topics:
   File: ossm-running-v2-same-cluster-as-v3-assembly
 - Name: Red Hat OpenShift Service Mesh and cert-manager
   File: ossm-cert-manager-assembly
+- Name: Using multiple control planes on a single cluster
```

Comment:
Why not rename this to "Deploying multiple service meshes on a single cluster", as that's the main point of this document. The fact that you need multiple control planes is a consequence, not the main point.

Reply:
Agreed - that's better. It looks like the change has been made - thank you!